frontend/frontend/src/components/TheNavBar/components/AccountDropdownContents.vue

```diff
@@ -127,15 +127,20 @@
       <b-form-group>
         <b-form-text style="font-size: 1.1em; display: flex; flex-direction: column;">
           <div class="first-row">
-            <div style="float: left; padding-top: 0.24em;">
-              User: 
+            <div style="padding-top: 0.24em; float: left;">
```

Comment:
```suggestion
            <div id="user">
```

Add css id below in <style>